test/e2e/apimachinery/resource_quota.go

```diff
@@ -1186,21 +1186,11 @@ var _ = SIGDescribe("ResourceQuota", func() {
 		})
 		framework.ExpectNoError(err, "failed to locate ResourceQuota %q in namespace %q", patchedResourceQuota.Name, ns)
 
-		err = wait.PollUntilContextTimeout(ctx, 5*time.Second, 5*time.Minute, true, func(ctx context.Context) (bool, error) {
```

Comment:
gce tests use a 3 mins value for min-sync period, kind uses kubeadm and uses a 12h value for the min-sync period.
My reading of the code is that the resource-quota-controller ignore changes to the status and reconciles it every ResyncPeriod, and this is a random value obtained from the kube-controller-manager flag min-sync-period and  is a value between min-sync-period and 2*min-sync-period.

Reply:
cc @Riaankl 
https://github.com/kubernetes/kubernetes/commit/168fa73b60b625ed1e69f2cd62f850ba46a24efc
(conformance promotion)

Reply:
if the resync period averages at 18h on kubeadm jobs and this part of the test is supposed to fail on the stock defaults, why is the test 99% green on kubeadm jobs?
https://testgrid.k8s.io/sig-cluster-lifecycle-kubeadm#kubeadm-kinder-latest

Reply:
that part I don't understand it too, that is why I asked for reviewers that know better the resource controller, but you can follow the codepath in https://github.com/kubernetes/kubernetes/issues/121911#issuecomment-1817473968

Reply:
/hold
we should be able to answer this question, there is a `pkg/controller/resourcequota/resource_quota_monitor.go` that I don't have clear what is doing

Reply:
/hold cancel
@neolit123 now I think I got it right, thanks for the great question

Reply:
updated the PR, it turns out the resyn period i other option https://github.com/kubernetes/kubernetes/issues/121911#issuecomment-1817593892